### DISCUSSION TITLE
Where is the load balancing target equivalent of this?

### DISCUSSION THREAD

**Discussion by jadedevin13** [rsocket/rsocket-js#181] [Q&A]
I can see that I can create a server using this. But how do I load balance to multiple server instances?

**Comment by viglucci** [comment]
Hi @jadedevin13 , there isn't much in the means of guidance in this area at the moment. Depending on the transport you choose (WebSocket, TCP, etc.) you will likely find solutions for load balancing, however, they will likely be ignorant of any RSocket-specific protocol concerns. If the intermediary doing the load balancing transparently passes messages between peers, ignorance of the protocol may be fine, however, I have not spent sufficient time exploring the topic so I am not sure.

I would expect the situation concerning load balancing and routing (ie: [rsocket-broker](https://github.com/rsocket-broker/rsocket-broker-spec)) to mature over time, but as there isn't much guidance to provide at the moment.
